tests/vmi_configuration_test.go

```diff
@@ -1121,12 +1121,23 @@ var _ = Describe("Configurations", func() {
 	})
 	Describe("[rfe_id:897][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance with CPU pinning", func() {
 		var nodes *kubev1.NodeList
+		var nodesWithCPUManager *kubev1.NodeList
+		var nodesWithoutCPUManager *kubev1.NodeList
+
 		BeforeEach(func() {
 			nodes, err = virtClient.CoreV1().Nodes().List(metav1.ListOptions{})
 			tests.PanicOnError(err)
 			if len(nodes.Items) == 1 {
 				Skip("Skip cpu pinning test that requires multiple nodes when only one node is present.")
 			}
+
+			nodesWithCPUManager, err := virtClient.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: v1.CPUManager + "=" + "true"})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
What you probably want is skipping the tests if there is no nod with the CPU manager there?